docs/integrations/ai-engines/rag.mdx

```diff
@@ -103,102 +99,130 @@ Now you can use the model to answer your questions.
-                        Context: {context}
-                        Question: {question}
-                        Helpful Answer:';
+CREATE DATABASE mysql_demo_db
+WITH ENGINE = 'mysql',
+PARAMETERS = {
+    "user": "user",
+    "password": "MindsDBUser123!",
+    "host": "db-demo-data.cwoyhfn6bzs0.us-east-1.rds.amazonaws.com",
+    "port": "3306",
+    "database": "public"
+};
+```
+
+Create a model using this engine and include the FORM clause:
+
+```sql
+CREATE MODEL rag_handler_db
+FROM mysql_demo_db 
+    (SELECT * FROM demo_fda_context LIMIT 2)
+PREDICT answer
+USING
+   engine="rag",
+   llm_type="openai",
+   vector_store_folder_name='test_db',
+   input_column='question';
 ```
 
-Now that the model is created, trained, and deployed, you can query for predictions.
+Now you can use the model to answer your questions.
 
 ```sql
- SELECT *
- FROM rag_openai_model
- WHERE question = 'what product is best for treating a cold?';
+SELECT *
+FROM rag_handler_db
+WHERE question='what product is best for treating a cold?';
 ```
 
-#### Writer
+On execution, we get:
+
+| answer                                                                                                                                                                                       | source_documents | question                                |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------|-----------------------------------------|
+|   ShopRite Arthritis Pain Acetaminophen is not specifically designed for treating a cold. It may help to temporarily relieve minor aches and pains associated with a cold, but it is not the best product for treating a cold. It is always best to consult with a doctor or pharmacist for the most appropriate medication for treating a cold. | `{"column":["full_ingredients","indications_and_usage","intended_purpose_of_product","active_ingredient"],"sources_content":["ShopRite Arthritis  ..."],"sources_document":["dataframe"],"sources_row":[1,1,1,1]}`   | what product is best for treating a cold? |
+
+
+
+
+### From VectorDB
+
+The following example utilize `rag_engine` to create a model with the `CREATE MODEL` statement and `ChromaDB` database as a knowlege base.
 
-Create the RAG engine providing credentials for LLM you want to use.
 
 ```sql
-CREATE ML_ENGINE rag_writer
+CREATE ML_ENGINE rag_engine
 FROM rag
 USING
-    writer_org_id = "value",
-    writer_api_key = "value";
+    openai_api_key = 'sk-xxx';
 ```
 
-Create a model and embed input data.
+Connect to ChromaDB:
 
 ```sql
- -- Create with writer and embed input data
- CREATE MODEL rag_writer_model
- FROM datasource
-    (SELECT * FROM table)
+CREATE DATABASE chromadb_datasource
```

Comment:
This vector db is not used further in CREATE MODEL.
The model is trained from mysl_demo_db as above.

Shall we remove this "### From VectorDB" section?